src/components/modeler/setUpSelectionBox.js

```diff
@@ -35,6 +44,32 @@ export default function setUpSelectionBox(setCursor, resetCursor, paperManager,
   let selectionBox;
 
   function shiftKeyDownListener(event) {
+
+    const highlightedShapes = store.getters.highlightedShapes;
```

Comment:
This is great for testing but I think in final-form they'd work nicely under icons in the toolbar.

Reply:
Yeah this needs to never go near actual humans :D